schemas/service.proto

```diff
@@ -33,6 +33,13 @@ message PutWfSpecRequest {
   string entrypoint_thread_name = 6;
   optional WorkflowRetentionPolicy retention_policy = 8;
   reserved 9;
+  optional AllowedUpdateType allowed_updates = 10;
+}
+
+enum AllowedUpdateType {
+  ALL = 0;
```

Comment:
Comments will be useful. And I believe this will add javadoc for the generated Java enum.